docs-authoring-pack/package.json

```diff
@@ -26,7 +26,11 @@
     "docsmsft.docs-preview",
     "docsmsft.docs-markdown",
     "docsmsft.docs-article-templates",
-    "streetsidesoftware.code-spell-checker"
+    "streetsidesoftware.code-spell-checker",
+    "docsmsft.docs-yaml"
   ],
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "vsce": "^1.58.0"
```

Comment:
Do we need this as a dependency? I though this was a global install

Reply:
Yeah, removed :)